docs/ja/core-concepts/ngsiv2-vs-ngsild.md

```
@@ -104,199 +106,109 @@ interface EntityDocument {
-  }'
-```
+プロトコル分離の導入前に作成されたエンティティ(つまり、データベースに `protocol` フィールドがないもの)は `'ngsild'` として扱われます。これらは NGSI-LD API 経由でのみアクセス可能です。
 
-**Retrieve the same entity with NGSI-LD:**
+### フェデレーションによるプロトコル間アクセス
 
-```bash
-curl http://localhost:3000/ngsi-ld/v1/entities/urn:ngsi-ld:Room:001 \
-  -H "Fiware-Service: demo"
-```
+プロトコル間の直接アクセスはサポートされていません。プロトコル間でエンティティにアクセスする必要がある場合は、**フェデレーション**(コンテキストソース登録)を使用して、1 つの GeonicDB インスタンスを別のプロトコルのコンテキストプロバイダーとして登録してください。詳細については、[フェデレーション](#federation)セクションを参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)